client/packages/system/src/Stock/Components/Repack/RepackEditForm.tsx

```diff
@@ -86,14 +86,15 @@ export const RepackEditForm: FC<RepackEditFormProps> = ({
         <InputWithLabelRow
           label={t('label.new-num-packs')}
           Input={
-            <PositiveNumberInput
+            <NonNegativeIntegerInput
               onChange={numberOfPacks => {
                 onInsert({
                   numberOfPacks,
                 });
               }}
               width={143}
               value={draft.numberOfPacks}
+              max={stockLine?.availableNumberOfPacks ?? 0}
```

Comment:
Disallow users from entering a number that is greater than what is available

Reply:
shouldn't it be `Math.floor(totalNumberOfUnits / newPackSize)`? or something like that..

Reply:
👀  new # of packs is going from newNum out of totalNum...

Reply:
> 👀 new # of packs is going from newNum out of stock line totalNum...